models/netsuite2/stg_netsuite2__transactions.sql

```diff
@@ -22,6 +22,7 @@ final as (
     
     select
         _fivetran_synced,
+        cast(_fivetran_synced as date) as _fivetran_synced_date,
```

Comment:
should this include a `date_trunc`, or does casting as a `date` automatically remove the time-part of the timestamp?

Reply:
The casting does remove the time-part, if you check out my validation from my ticket that shows the result of the cast. The date_trunc is the one that leaves the time-part I believe.